Solutions/CohesitySecurity/Data Connectors/helios_connector.json

```diff
@@ -0,0 +1,117 @@
+                }],
+                "connectivityCriteria": [{
+                    "type": "SentinelKindsV2",
+                    "value": []
+                }],
+                "availability": {
+                    "status": 1,
+                    "isPreview": true
+                },
+                "permissions": {
+                    "resourceProvider": [{
+                            "provider": "Microsoft.OperationalInsights/workspaces",
+                            "permissionsDisplayText": "read and write permissions are required.",
+                            "providerDisplayName": "Workspace",
+                            "scope": "Workspace",
+                            "requiredPermissions": {
+                                "write": true,
+                                "read": true,
+                                "delete": true
+                            }
+                        },
+                        {
+                            "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                            "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key)",
+                            "providerDisplayName": "Keys",
+                            "scope": "Workspace",
+                            "requiredPermissions": {
+                                "action": true
+                            }
+                        }
+                    ],
+                    "customs": [{
+                        "name": "Cohesity Security Alerts API credentials",
+                        "description": "**HeliosAPIBearerToken** is required for REST API. [See the documentation to learn more about API](https://api.helios.com/web#authentication). Check all [requirements and follow the instructions](https://api.Cohesity.com/web#authentication) for obtaining credentials."
+                    }]
+                },
+                "instructionSteps": [{
+                    "title": "Connect Cohesity Security Alerts to Microsoft Sentinel",
+                    "description": "Enable Cohesity Security Alerts Logs.",
+                    "instructions": [{
+                        "parameters": {
+                            "enable": "true"
+                        },
+                        "type": "APIKey"
+                    }]
+                }]
+            },
+            "pollingConfig": {
+                "auth": {
+                    "authType": "APIKey",
+                    "APIKeyName": "apiKey"
+                },
+                "request": {
+                    "apiEndpoint": "https://helios.cohesity.com/mcm/alerts?maxAlerts=10&alertCategoryList=kSecurity&alertStateList=kOpen",
```

Comment:
For our internal testing, apiEndpoint URL should be: helios-test2.cohesitycloud.co 
For packaging and publication: helios.cohesity.com is ok.